recirq/third_party/quaff/basis_change_test.py

```diff
@@ -1,7 +1,7 @@
 import cirq
 import numpy as np
 import pytest
-import quaff
+from recirq.third_party import quaff as quaff
```

Comment:
nit - `as quaff` is redundant -
```suggestion
from recirq.third_party import quaff
```
also please fix the same import in comb_test.py, cz_layer_test.py, indexing_test.py, linalg_test.py, random_test.py

Reply:
Whoops, thanks.